lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -675,6 +697,65 @@ def test_group_info_in_ajax_response(self, mock_request):
         )
 
 
+@patch('requests.request', autospec=True)
+class ForumFormDiscussionContentGroupTestCase(ForumsEnableMixin, ContentGroupTestCase):
+    shard = 4
+
+    @patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
+    def setUp(self):
+        super(ForumFormDiscussionContentGroupTestCase, self).setUp()
+
+    def assert_response(self, response, expected_discussion_threads):
+        discussion_data = json.loads(response.content)['discussion_data']
+        self.assertEqual(len(discussion_data), expected_discussion_threads)
+
+    def call_view(self, mock_request, user):
+        mock_request.side_effect = make_mock_request_impl(
+            course=self.course,
+            text="dummy content",
+            thread_id="test_thread_id",
+            is_cohorted=True
+        )
+        self.client.login(username=user.username, password='test')
+        return self.client.get(
+            reverse("forum_form_discussion", args=[unicode(self.course.id)]),
+            HTTP_X_REQUESTED_WITH="XMLHttpRequest"
+        )
+
+    def test_community_ta_user(self, mock_request):
+        """
+        Verify that Community TA has access to all threads regardless
+        of cohort.
+        """
+        response = self.call_view(
+            mock_request,
+            self.community_ta
+        )
+        self.assert_response(response, 4)
+
+    def test_alpha_cohort_user(self, mock_request):
+        """
+        Verify that alpha cohort user has access to alpha cohort threads
+        and non-cohorted threads.
+        """
+        response = self.call_view(
+            mock_request,
+            self.alpha_user
+        )
+        self.assert_response(response, 3)
```

Comment:
What are the `alpha` and `beta` cohort users? and why we are expecting a length of 4 in tests without users and 3 with users? where in tests it is specified that a user (`alpha` or `beta`) must have access to particular number of discussion threads?